about.html

```diff
@@ -0,0 +1,128 @@
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="./about.css">
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <div class="nav-socials">
+          <img src="./images/facebook.png" alt="">
+          <img src="./images/twitter.png" alt="">
+          <a href="#">English</a>
+          <a href="#">My page</a>
+          <a href="#">Logout</a>
+        </div>
+        <div class="navbar">
+          <div class="navbar-logo">
+            <label>
+              <a href="./index.html">
+                DEVSPARK
+              </a>
+            </label>
+          </div>
+          <div class="navbar-mobile-close">
+            <img src="./images/Icon-Cancel.svg" alt="">
+          </div>
+          <div class="navbar-menus">
+            <ul>
+              <li>
+                <a href="./index.html">Home</a>
+              </li>
+              <li>
+                <a href="#">Program</a>
+              </li>
+              <li>
+                <a href="#">Join</a>
+              </li>
+              <li>
+                <a href="#">Sponsor</a>
+              </li>
+              <li>
+                <a href="#">News</a>
+              </li>
+              <button>CC Campaign</button>
+            </ul>
+          </div>
+          <div class="navbar-mobile-open">
+            <img src="./images/Icon - Menu.svg" alt="">
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="about-home">
+      <div class="about-home-text">
+        <h2>"Be among the Best Developers"</h2>
+        <h1>Get Good at Building Websites</h1>
+        <p class="about-home-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem enim obcaecati praesentium veniam nisi suscipit error, modi quisquam eaque fugit asperiores at itaque amet blanditiis mollitia pariatur?</p>
+        <div class="date-time">
+          <h3>Please contact us per Email for any further questions about DEVSPARK</h3>
+          <p><a href="mailto:newtonombese1@gmail.com">info@devspark.com</a></p>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-story">
+      <h2>Devspark Major Coding Summit 2023</h2>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      <img src="./images/IMG_20230227_082950_605.jpg" alt="">
+    </section>
+
+    <section class="about-events">
+      <h2>See Devspark Previous Coding Summits</h2>
+      <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit.</p>
+      <div class="event-images">
+        <div class="image-1">
+          <div class="image-text-1">
+            <h2>2022</h2>
+            <p>Devspark Summit @ Moringa</p>
+          </div>
+        </div>
+        <div class="image-2">
+          <div class="image-text-2">
+            <h2>2023</h2>
+            <p>Devspark Summit @ iHub</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [x] Good job but your about page design is a bit different from the original design :cry: You can check the difference with the screenshots I'm sharing with you below :arrow_down:  Please make sure to fix it in order to make your design work as expected :+1: 

#### Your design
![about](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/0f5f1f90-988a-42bd-ba0d-bf65d6c833f7)


#### Original design
![about_design](https://user-images.githubusercontent.com/6615394/210892739-9a698a5d-36f7-4363-a666-7890975a02c3.jpg)

Reply:
Fixed